pkg/compute/compute_test.go

```diff
@@ -30,19 +30,81 @@ func TestInit(t *testing.T) {
 		t.Skip("Set TEST_COMPUTE_INIT to run this test")
 	}
 
-	if err := os.Chdir("testdata/init"); err != nil {
-		t.Fatal(err)
-	}
-
 	for _, testcase := range []struct {
-		name       string
-		args       []string
-		wantError  string
-		wantOutput string
+		name            string
+		args            []string
+		wantFiles       []string
+		unwantedFiles   []string
+		wantError       string
+		wantOutput      []string
+		manifestContent string
 	}{
-		// TODO(pb): wait until package template repo is public
```

Comment:
☝️

Reply:
I'm slightly confused, the repo is now public, hence why I've written these tests?

Reply:
I know, I was just having a little laugh, please ignore :)